.github/PULL_REQUEST_TEMPLATE

```diff
@@ -17,6 +17,8 @@ Mention the new unit tests added
 Was any manual testing needed? If yes, is there a need to add integration tests for that?
 -->
 
+### Does your PR have changes in callhome/yugabyted payloads? If so, is the payload version incremented?
```

Comment:
nit: maybe bold few words in the text.

Reason being our template is quiet big and there are chances of  missing to read it.

Reply:
The whole test already comes up as a header (at the same level as the existing high level questions), so bold isn't really possible here.